pkg/rpc/response/result/application_log.go

```diff
@@ -80,30 +72,14 @@ func (l *ApplicationLog) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-// StateEventToResultNotification converts state.NotificationEvent to
-// result.NotificationEvent.
-func StateEventToResultNotification(event state.NotificationEvent) NotificationEvent {
-	seen := make(map[stackitem.Item]bool)
-	item := smartcontract.ParameterFromStackItem(event.Item, seen)
-	return NotificationEvent{
-		Contract: event.ScriptHash,
-		Name:     event.Name,
-		Item:     item,
-	}
-}
-
 // NewApplicationLog creates a new ApplicationLog wrapper.
 func NewApplicationLog(appExecRes *state.AppExecResult) ApplicationLog {
-	events := make([]NotificationEvent, 0, len(appExecRes.Events))
-	for _, e := range appExecRes.Events {
-		events = append(events, StateEventToResultNotification(e))
-	}
 	return ApplicationLog{
```

Comment:
Do we need a `result.ApplicationLog` then? Maybe `state.AppExecResult` could be JSONized straight to the format required?